client_interface.py

```diff
@@ -49,18 +51,19 @@ def send_to_server(self, state):
             async with session.ws_connect('http://localhost:8080/interface/') as self.ws:
                 async for msg in self.ws:
                     # Cycle "for" is finished when client disconnect from server
-                    if msg.type == aiohttp.WSMsgType.TEXT:
-                        if msg.data.startswith("robot"):
-                            robot = msg.data
-                            print(robot)
-                        if msg.data.startswith("cards"):
-                            cards = msg.data
-                            print(cards)
-                        else:
-                            print(msg.data)
+                    message = msg.json(loads=json.loads)
+                    # print(message)
+                    if "game_state" in message.keys():
```

Comment:
to je co za metodu message.keys? proč tam nemůže být `if message["game_state"]:` jako je v client_receiver? v čem je rozdíl?

Reply:
Zkus to a uvidíš. `.keys()` je způsob jak se dostat ke klíčům slovníku, na kterém to aplikuji. Najdeš to v materiálech PyLadies o slovnících.

Reply:
Jednodušší způsob jak zjistit jestli slovník má záznam s daným klíčem je `"game_state" in message`.